src/main/java/s8u/studies/myapplication/model/Pokemon.kt

```diff
@@ -0,0 +1,18 @@
+package s8u.studies.myapplication.model
+
+import com.google.gson.annotations.SerializedName
+
+open class Pokemon(
```

Comment:
Essa classe precisa ser open? 

Só mantemos open quando queremos fazer herança, se não for o caso, o que acha de remover o open?